tlf/README

```diff
@@ -0,0 +1,3 @@
+Top-level folder (TLF)-related types and functions for KBFS.
```

Comment:
Can you add a link/mention of this to the top-level `README` as well?  Maybe for the other packages you've added recently as well.